td-agent/Rakefile

```diff
@@ -868,13 +866,23 @@ class BuildTask
 
     relative_path = File.join(File.basename(File.split(DOWNLOADS_DIR)[0]), File.split(DOWNLOADS_DIR)[1])
     puts "bundle config cache path: <#{relative_path}>"
-    sh(bundle_command, "config", "set", "--local", "cache_path", relative_path)
-    sh(bundle_command, "package", "--no-install")
+    gem_home = ENV["GEM_HOME"]
+    ENV["GEM_HOME"] = gem_staging_dir
+    sh(bundle_command, "_2.2.0.rc.1_", "config", "set", "--local", "cache_path", relative_path)
+    sh(bundle_command, "_2.2.0.rc.1_", "package", "--no-install")
+    ENV["GEM_HOME"] = gem_home
 
+    Dir.glob("**/*.gem").sort.each do |gem_path|
+      puts "fetched gem path: <#{gem_path}>"
```

Comment:
nits:
How about using [logger](https://docs.ruby-lang.org/ja/latest/class/Logger.html) to specify log level in the later PR?

Reply:
It seems reasonable!